### DISCUSSION TITLE
Is there an estimated date as to when 10.2 will be released?

### DISCUSSION THREAD

**Discussion by tbeech** [realm/realm-dotnet#2440] [General]
Just looking to see if there's an estimate as to when 10.2 is going to be released?

**Comment by nirinchev** [comment]
Likely within a week. Is there something in particular you're looking for from 10.2.0? We do have a beta out, although that's a few weeks old now and doesn't include some of the latest fixes.

**Reply by tbeech** [reply]
That's perfect. There are some enhancements with query support that I'm looking for, but nothing that can't wait for the actual release. I was hoping that 10.2 was getting close to release!
